bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss

```diff
@@ -19,8 +19,8 @@
   --indicator-padding: .425rem;
 
   //used to center indicator icons in Edge
-  --indicator-padding-right: 1.2em;
-  --indicator-padding-left: 0.175em;
+  --indicator-padding-right: .55em;
```

Comment:
What is the reasoning behind these changes? What was different in Edge? Looking at the issue I expected to see some `-ms-` prefixes